source/nodejs/adaptivecards-site/themes/adaptivecards/source/js/script.js

```diff
@@ -475,10 +475,18 @@ $(function () {
 
 	});
 
+	$("button.copy-data").click(function (e) {
+		var content = $(this).parent().next("pre").text();
+		copyToClipboard(content);
+		$(this).append("<span role='status' aria-label='Sample data copied successfully' id='dataCopySuccess' />");
+		$(this).focus();
```

Comment:
>$(this).focus() [](start = 2, length = 15)

How does this effect where the keyboard focus lands after invoking? Playing around with both your version and the shipped site the keyboard focus wanders off someplace weird and inconsistent for me in both cases, so it doesn't look like this makes things any more broken than the already are (pretty broken). That said, what the is expectation with how the keyboard focus finds its way back since this line explicitly moves the focus to this span?

Reply:
the shipped site has a bug with templated samples due to both copy buttons having the `copy-code` class. I noticed that and fixed it in this change. 

as for the `focus()` call itself, we're re-focusing `$(this)`, which is `$("button.copy-data")` or `$("button.copy-code")`, and not the newly-added child.

Reply:
I'm still seeing weirdness here, the narrator focus often wanders off to seemingly random places after clicking copy. That said, I'm seeing it on the shipped site too, so it doesn't seem to be related to this change.